easybuild/easyconfigs/a/ArmForge/ArmForge-22.1.2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'Binary'
```

Comment:
@boegel I noticed there are some other open PRs for ArmForge/Arm-Forge, including https://github.com/easybuilders/easybuild-easyconfigs/pull/15252, which makes use of the Allinea easyblock. As ArmForge seems to be a rebrand of the Allinea tools, and it still supports these site config files and templates and also still requires a license, things which are all implemented in that easyblock, wouldn't it be better to use (a slightly modified/renamed version of) that easyblock?